taui/src/components/edit-profile.js

```diff
@@ -248,43 +191,8 @@ class EditProfile extends PureComponent<Props> {
       this.setState({errorMessage: ''})
     }
 
-    if (!isAnonymous) {
-      // Create user login account if new email set
-      if (profile.key.indexOf('_') < 0 && profile.clientEmail && !profile.clientInviteSent) {
-        console.log('go create user account')
-
-        // verify it first
-        if (profile.clientEmail && EMAIL_REGEX.test(profile.clientEmail)) {
-          this.createClientAccount(profile.key).then(createResponse => {
-            if (createResponse === 'created') {
-              console.log('user account has been created (or already exists)')
-              profile.clientInviteSent = true
-              // Save to S3 once user account creation succeeded
-              this.saveToS3(profile.key, profile, isCounselor, this.props.changeUserProfile)
-            } else if (createResponse === 'exists') {
-              console.log('user account already confirmed for voucher number; save new profile')
-              profile.clientInviteSent = true
-              profile.clientAccountConfirmed = true
-              this.saveToS3(profile.key, profile, isCounselor, this.props.changeUserProfile)
-            } else {
-              console.error('Failed to create user account')
-              // Do not save to S3 so counselor can see the error and stay on profile page
-            }
-          })
-        } else {
-          console.warn('does not look like an email', profile.clientEmail)
-          this.setState({errorMessage: t('Profile.ClientEmailError')})
-        }
-      } else {
-        console.log('Do not need to create client account; it already exists')
-        // go save to s3 without attempting to first create a user account
-        this.saveToS3(profile.key, profile, isCounselor, this.props.changeUserProfile)
-      }
-    } else {
-      // Do not attempt to write anonymous profile to S3
-      this.props.changeUserProfile(profile)
-      this.props.history.push('/map')
-    }
+    this.props.handleProfileChange(profile)
+    this.props.history.push('/map')
   }
 
   getUserDataVoucherNumber (userData: any): string {
```

Comment:
I think we can get rid of `getUserDataVoucherNumber` now since it's not used by anything